lighthouse-core/test/gather/computed/pushed-requests-test.js

```diff
@@ -7,15 +7,17 @@
 
 /* eslint-env mocha */
 
-const GathererClass = require('../../../gather/computed/pushed-requests');
+const Runner = require('../../../runner.js');
```

Comment:
I can update this one too, but I don't even know why we have this gatherer and really am tempted to just delete it :)